src/lib/request.ts

```diff
@@ -102,3 +92,47 @@ export async function request(
 
   return json.values;
 }
+
+function handleError(errorResponse: string | object) {
```

Comment:
You could add a return type of `never` to this function.
https://www.typescriptlang.org/docs/handbook/2/functions.html#never